src/gsy_e/models/strategy/profile.py

```diff
@@ -50,12 +50,7 @@ def read_or_rotate_profiles(self, reconfigure=False):
         else:
             profile = self.profile
 
-        profile = global_objects.profiles_handler.rotate_profile(
+        self.profile = global_objects.profiles_handler.rotate_profile(
             profile_type=self.profile_type,
             profile=profile,
             profile_uuid=self.input_profile_uuid)
-
-        if self.profile_type == InputProfileTypes.IDENTITY:
-            self.profile = convert_identity_profile_to_float(profile)
```

Comment:
Just for my understanding, the only reason for these lines was to support profiles with type `Dict[datetime, str]` ? If yes I am glad that we no longer accept such profiles

Reply:
Exactly. When I found that out, I was also really relieved to remove it. :-)